core/tests/protractor_utils/users.js

```diff
@@ -85,6 +85,11 @@ var logout = async function() {
 // that this will fail if the user already has a username.
 var _completeSignup = async function(username) {
   await waitFor.pageToFullyLoad();
+  let cookieBanner = element(
```

Comment:
Can we change this to `cookieBannerAcceptButton`?

Reply:
done.